corehq/apps/translations/app_translations/upload_app.py

```diff
@@ -192,24 +237,22 @@ def _process_rows(app, sheet_name, rows, names_map, lang=None):
     )]
 
 
-def _check_for_workbook_error(app, workbook, headers):
-    if len(headers) == 1 and len(workbook.worksheets) > 1:
+def _check_workbook_length(workbook, lang):
```

Comment:
aha, that's what length meant. :) I could not connect length to number of worksheets

nit: `_ensure_correct_number_of_worksheets` or `_check_number_of_worksheets`